src/ISA/ISA/ARCTypes/CompositeHeader.fs

```diff
@@ -1,30 +1,177 @@
 ﻿namespace ISA
 
+open Fable.Core
+
+// TODO: Do we need FreeText of string on this one too? Do we want to allow other IOTypes?
```

Comment:
Hmm good question. But since we included the "Input" and "Output" header logic in the ISA-XLSX this might be necessary.